sdk_container/src/third_party/coreos-overlay/coreos-base/misc-files/files/ntp/ntp.conf

```diff
@@ -10,6 +10,11 @@
 # Common pool for random people
 #server pool.ntp.org
 
+# AWS Time Sync Service.
+server 169.254.169.123 prefer iburst minpoll 4 maxpoll 4
+# Comment the above line and uncomment the below one on AWS IPv6 instances:
+# server fd00:ec2::123 prefer iburst minpoll 4 maxpoll 4
```

Comment:
Can we have this enabled by default? It would not be working in the IPv4-only case but reading the commit message that sounds to be ok, or?